userguide/content/en/docs/Language/_index.md

```diff
@@ -98,10 +98,14 @@ If you configure more than one language in `config.toml`, the Docsy theme adds a
 
 ## Internationalization bundles
 
-All UI strings (text for buttons etc.) are bundled inside `/i18n` in the theme, with a `.toml` file for each language.
+All UI strings (text for buttons, repository links, etc.) are bundled inside `/i18n` in the theme, with a `.toml` file for each language.
 
-If your chosen language isn't currently in the theme and you create your own `.toml` file for all the common UI strings (for example, if you translate the UI text into Japanese and create a copy of `en.toml` called `jp.toml`), we recommend you do this **in the theme** rather than in your own project, so it can be reused by others. Any additional strings or overridden values can be added to your project's `/i18n` folder.
+If your chosen language isn't currently in the theme and you create your own `.toml` file for all the common UI strings (for example, if you translate the UI text into Japanese and create a copy of `en.toml` called `jp.toml`), we recommend you do this **in the theme** rather than in your own project, so it can be reused by others. 
```

Comment:
> we recommend you do this **in the theme** rather than in your own project, so it can be reused by others.

I see the point of local reuse. However, why don't we take the opportunity and encourage users to contribute their translations to the community. Something like:

we recommend you do this **in the theme** rather than in your own project. Consider opening a new [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) to contribute your translation to the docsy community.

Reply:
That's actually what I meant but your wording is clearer. Done!